packages/components/src/text-highlight/test/index.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import TextHighlight from '..';
+
+const getMarks = ( container: Element ) =>
+	Array.from( container.querySelectorAll( 'mark' ) );
```

Comment:
TIL the `mark` role is [not officially in the spec](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/mark_role#specifications) yet, thus not supported by `getByRole`!

Reply:
TIL too! Should we add a comment about this, saying that the code should be updated to use its intrinsic `role` once the spec update lands?

Reply:
Added a comment about it.